move-vm-backend/src/deposit.rs

```diff
@@ -0,0 +1,37 @@
+#[cfg(not(feature = "std"))]
+use alloc::{borrow::ToOwned, string::ToString, vec};
+use lazy_static::lazy_static;
+use move_core_types::{
+    account_address::AccountAddress,
+    ident_str,
+    language_storage::{StructTag, TypeTag},
+};
+use serde::{Deserialize, Serialize};
+
+/// Mirroring structure of Move::deposit::Deposit
+/// Designed for bridging move transfer and native tokens of substrate
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub(crate) struct Deposit {
+    destination: AccountAddress,
+    amount: u128,
+}
+
+impl Into<(AccountAddress, u128)> for Deposit {
+    fn into(self) -> (AccountAddress, u128) {
+        (self.destination, self.amount)
+    }
+}
+
+lazy_static! {
+    pub(crate) static ref DEPOSIT_TEMPLATE: StructTag = StructTag {
+        address: DEPOSIT_CODE_ADDRESS.clone(),
+        module: ident_str!("DepositModule").to_owned(),
+        name: ident_str!("Deposit").to_owned(),
+        type_params: vec![TypeTag::U8],
+    };
+    /// Publisher address of DepositModule
+    pub static ref DEPOSIT_CODE_ADDRESS: AccountAddress = AccountAddress::from_hex_literal("0x42").unwrap();
+    /// Actual bytes of DepositModule module for use with pallet/Mvm
+    pub static ref MOVE_DEPOSIT_MODULE_BYTES: [u8; 299] =
```

Comment:
That's something that is not compiling on my computer when I try to compile the whole node.

```
error[E0308]: mismatched types
    --> /home/asmie/.cargo/git/checkouts/substrate-move-c751d829341564ee/bc50700/move-vm-backend/src/deposit.rs:36:9
     |
  35 |     pub static ref MOVE_DEPOSIT_MODULE_BYTES: [u8; 299] =
     |                                               --------- expected `[u8; 299]` because of return type
  36 |         include_bytes!("../../DepositModule.mv").to_owned();
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected an array with a fixed size of 299 elements, found one with 331 elements
```

Reply:
Made it into `Vec` for flex sizes